### PR TITLE
Bugfix/zcs 1469

### PR DESCRIPTION
**Pull request by haitran1696**
@silentsakky Please help me to review the code

### CODE REVIEW

**Review by fatkudu — APPROVED** [review]
You could squash these 2 changes before merging perhaps.

File: src/java/com/zimbra/cert/GetCert.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 public class GetCert extends AdminDocumentHandler {
     final static String CERT_TYPE_STAGED= "staged" ;
     final static String CERT_TYPE_ALL = "all" ;
-    final static String [] CERT_TYPES = {"ldap", "mailboxd", "mta", "proxy"};
+    final static String [] CERT_TYPES = {"imapd","ldap", "mailboxd", "mta", "proxy"};
```
Comment (plobbes) [inline src/java/com/zimbra/cert/GetCert.java]: nit: space after ','

**Review by silentsakky — APPROVED** [review]

**Review by fatkudu — CHANGES_REQUESTED** [review]
Please squash the commits.  We really don't need 3 commits for this small change.

**Comment by grishick** [comment]
@haitran1696 please e-sign the CLA

**Comment by bluesea1696** [comment]
@grishick I signed the CLA but I see that license/cla is not signed yet. 
sign cla
cla

**Comment by haitran1696** [comment]
@grishick I have signed the CLA but the status is still pending. I also rechecked it. 
sign cla
cla

**Comment by haitran1696** [comment]
Hi @grishick, I signed the CLA successfully

**Review by fatkudu — APPROVED** [review]

**Review by plobbes — APPROVED** [review]

**Review by gordyt — APPROVED** [review]

**Review by grishick — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Comment by silentsakky** [comment]
Ticket closed as Won't Fix

**Pull request closed without merging**